pkgs/by-name/ro/rofi-blezz/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, cairo
+, pkg-config
+, rofi-unwrapped
+, wrapGAppsHook3
+}:
+
+stdenv.mkDerivation {
+  pname = "rofi-blezz";
+  version = "unstable-2022-09-07";
+
+  src = fetchFromGitHub {
+    owner = "davatorium";
+    repo = "rofi-blezz";
+    rev = "3a00473471e7c56d2c349ad437937107b7d8e961";
+    sha256 = "sha256-hY5UA7nyL6QoOBIZTjEiR0zjZFhkUkRa50r5rVZDnbg=";
+  };
+
+  patches = [
+    ./0001-Patch-plugindir-to-output.patch
+    ./0002-Patch-add-cairo.patch
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+    wrapGAppsHook3
```

Comment:
wrappers are wrapping executables not libraries, so what does `wrapGAppsHook3` do here?

Reply:
Yeah it is not required. Based my package on `rofi-top` that uses it. I have removed the wrapper and `rofi-blezz` works as expected.